shared/charts/cert-manager/templates/00-namespace.yaml

```diff
@@ -1,9 +0,0 @@
-{{ if .Values.createNamespaceResource }}
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: {{ .Release.Namespace | quote }}
-  labels:
-    name: {{ .Release.Namespace | quote }}
-    certmanager.k8s.io/disable-validation: "true"
```

Comment:
Would undeleting this file help with the manual label command issue?

Reply:
In our case this does not matter, as we deploy to kube-system. I've removed it as the original chart removed it as well.